install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2

```diff
@@ -222,22 +222,22 @@
   /var/www/securedrop/source_app/session_manager.py r,
   /var/www/securedrop/source_app/utils.py r,
   /var/www/securedrop/source_templates/base.html r,
+  /var/www/securedrop/source_templates/codename.html r,
   /var/www/securedrop/source_templates/error.html r,
-  /var/www/securedrop/source_templates/first_submission_flashed_message.html r,
   /var/www/securedrop/source_templates/flashed.html r,
+  /var/www/securedrop/source_templates/flash_message.html r,
   /var/www/securedrop/source_templates/footer.html r,
   /var/www/securedrop/source_templates/generate.html r,
   /var/www/securedrop/source_templates/index.html r,
   /var/www/securedrop/source_templates/locales.html r,
   /var/www/securedrop/source_templates/login.html r,
   /var/www/securedrop/source_templates/logout.html r,
   /var/www/securedrop/source_templates/lookup.html r,
-  /var/www/securedrop/source_templates/next_submission_flashed_message.html r,
   /var/www/securedrop/source_templates/notfound.html r,
   /var/www/securedrop/source_templates/session_timeout.html r,
   /var/www/securedrop/source_templates/tor2web-warning.html r,
-  /var/www/securedrop/source_templates/utils.html r,
   /var/www/securedrop/source_templates/use-tor-browser.html r,
+  /var/www/securedrop/source_templates/utils.html r,
   /var/www/securedrop/source_templates/why-public-key.html r,
   /var/www/securedrop/static/.webassets-cache/** rw,
   /var/www/securedrop/static/css/font-awesome.css r,
```

Comment:
not introduced in this release, but looks like this and static/css/normalize.css has been gone for a long time!